README.md

```diff
@@ -241,15 +241,61 @@ CKB> account list
   ledger_id: 0x69c46b6dd072a2693378ef4f5f35dcd82f826dc1fdcc891255db5870f54b06e6
 ```
 
-The `ledger_id` shown is the public key hash for the path m/44'/309', which is the root Nervos path. Remember the `ledger\_id` given above for the latest commands. It will be
-used for ```<ledger-id>``` later on.
+The `ledger_id` shown is the public key hash for the path m/44'/309', which is the root Nervos path. the `ledger\_id` will be
+used for ```<ledger-id>``` argument in the `account import` command as described below.
 
-### Get Public Key ###
+If you have already imported the Ledger account, then `account list` command will instead give the account details.
+This will be shown even if the device is not connected.
 
-Get the public key:
+``` sh
+CKB> account list
+- "#": 0
+  account_source: ledger hardware wallet
+  address:
+    mainnet: ckb1qyqry754h4tevmngdll9jrpnrnfhqqhpcccskdl3en
+    testnet: ckt1qyqry754h4tevmngdll9jrpnrnfhqqhpcccstgpw40
+  lock_arg: 0x327a95bd57966e686ffe590c331cd37002e1c631
+  lock_hash: 0xc27b9ad3414cf5b1720713663d5f754e8968793f2da90b6428feb565bf94de4e
+```
+
+## Import Ledger Wallet account ###
+
+Use the `account import --ledger <ledger_id>` command to import the account to the `ckb-cli`.
+You will receive a confirmation prompt on the device which should say `Import Account` and `Account 1`.
```

Comment:
Perhaps it is worthwhile to note that this will happen in two prompts. "... which should say 'Import Account' followed by 'Account 1'."